problems/[230105] 82. Remove Duplicates from Sorted List II.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/remove-duplicates-from-sorted-list-ii/description/
+ *
+ * Definition for singly-linked list.
+ * function ListNode(val, next) {
+ *     this.val = (val===undefined ? 0 : val)
+ *     this.next = (next===undefined ? null : next)
+ * }
+ */
+/**
+ * @param {ListNode} head
+ * @return {ListNode}
+ */
+const deleteDuplicates = function (head) {
+  let resultNode = head;
+  let result = resultNode;
+
+  let isDuplicated = false;
+
+  while (resultNode.next) {
+    if (resultNode.val === resultNode.next) {
```

Comment:
문제의 전체적인 풀이와 관계없이, 만약 작성해주신 것과 같은 연산을 하고싶으셨다면 아래와 같은 형태가 되어야 할 것 같습니다!
왜냐하면 `resultNode`는 현재 `ListNode`의 인스턴스이고, 마찬가지로 `resultNode`의 `next` 프로퍼티에 담긴 값도 `ListNode`의 인스턴스이기 때문입니다 😃 

```suggestion
    if (resultNode.val === resultNode.next.val) {
```